common/priorities/priority_util.go

```diff
@@ -0,0 +1,73 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
```

Comment:
this is brand new, I think we can go with 2025 Temporal only

Reply:
👍  I just ran `make copyright`